internal/autoconfig/stream_manager.go

```diff
@@ -307,12 +307,6 @@ func (s *StreamManager) consumeStream(stream *es.Stream) {
 			if shouldRestart {
 				stream.Restart()
 			}
-
```

Comment:
Now stream manager doesn't need to worry about keys, which was entirely outside its normal purview.